self/chain.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "self",
+  "status": "live",
+  "website": "https://selfchain.xyz",
+  "network_type": "mainnet",
+  "pretty_name": "Self Chain",
+  "chain_id": "self-1",
+  "bech32_prefix": "self",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
https://github.com/satoshilabs/slips/pull/1751